toolkit/offchain/src/governance.rs

```diff
@@ -193,6 +193,12 @@ fn parse_simple_at_least_n_native_script(
 				key_hashes: keys,
 			}))
 		},
+		Some(OgmiosNativeScript::Signature { from: key_hash }) => {
```

Comment:
This isn't really AtLeastN. I don't know if you want to support it? I don't see a reason. Did you intended to use it in init governance with single key?

Doesn't cost very much to have it, but it is a bit confusing.